source/extensions/tracers/xray/tracer.cc

```diff
@@ -173,6 +176,8 @@ void Span::setTag(absl::string_view name, absl::string_view value) {
   constexpr auto HttpUserAgent = "user_agent";
```

Comment:
I know this isn't part of your change here, but perhaps we can look into using the built in headers where this value likely already exists.  If we have to take on a new module dependency here, then ignore me :)